torchsynth/module.py

```diff
@@ -363,11 +398,10 @@ def _ramp(
         method works by tilting and clipping ramps between 0 and 1, then
         applying some scaling factor (`alpha`).
 
-        Parameters
-        ----------
-        duration T: Length of the ramp in seconds
-        start Optional[T]: Initial delay of ramp in seconds. Defaults to None (0 start)
```

Comment:
Also don't need to specify default values -- this shows up in the method template in the docs